customize-homepage.md

```diff
@@ -0,0 +1,82 @@
+# Customize your home page
+
+When building your portfolio website, the first step is to set up and personalize your homepage. The homepage is your chance to make an excellent first impression and convey the core message of your portfolio. So your homepage should include the following features:
+
+1. Introduction: A concise introduction captures visitors' attention.
+2. Biography: Include a brief biography that introduces yourself. This section should mention your name, role, expertise, and unique qualities.
+3. Hero Image: This may be a professional headshot or graphical representation that showcases your work and adds visual appeal.
+4. Call to Action (CTA): Incorporate a CTA that guides visitors to take a specific action, such as "View Portfolio," "Hire Me," or "Learn More."
+5. Downloadable Resume
+
+In this section, you'll learn how to add features 1 to 4 to your homepage. We'll cover adding your resume or CV later in the tutorial.
+
+Now, modify your `home/models` to include the following:
+
+```python
+from django.db import models
+
+from wagtail.models import Page
+from wagtail.fields import RichTextField
+from wagtail.admin.panels import FieldPanel
+from wagtail.admin.panels import FieldPanel, MultiFieldPanel
+
+
+class HomePage(Page):
+    # Hero section of HomePage
+    image = models.ForeignKey(
+        "wagtailimages.Image",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+        help_text="Homepage image",
+    )
+    hero_text = models.CharField(
+        blank=True,
+        max_length=255, help_text="Write an introduction for the site"
+    )
+    hero_cta = models.CharField(
+        blank=True,
+        verbose_name="Hero CTA",
+        max_length=255,
+        help_text="Text to display on Call to Action",
+    )
+    hero_cta_link = models.ForeignKey(
+        "wagtailcore.Page",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+        verbose_name="Hero CTA link",
+        help_text="Choose a page to link to for the Call to Action",
+    )
+
+    # Body section of the HomePage
+    body = RichTextField(blank=True)
+
+    content_panels = Page.content_panels + [
+        MultiFieldPanel(
+            [
+                FieldPanel("image"),
+                FieldPanel("hero_text"),
+                FieldPanel("hero_cta"),
+                FieldPanel("hero_cta_link"),
+            ],
+            heading="Hero section",
+        ),
+        FieldPanel('body'),
+    ]
+```
+
+You might already be familiar with the different parts of your Homepage model. The `image` field is a `ForeignKey` referencing Wagtail's built-in Image model for storing images. Similarly, `hero_cta_link` is a `ForeignKey` to `wagtailcore.Page`, which is a page instance. 
```

Comment:
`wagtailcore.Page` is a class, not an instance. It’s the base class for all other page types in Wagtail (as in – all inherit from it). For example – `class BlogPage(Page)` – that `Page` is `wagtailcore.Page`.

We import it as `from wagtail.models import Page` because it’s where it is situated in the code - and that part of the code is registered in Django as the "wagtailcore" app.

```suggestion
Similarly, `hero_cta_link` is a `ForeignKey` to `wagtailcore.Page`, which is the base class for all page types, therefore allowing us to choose a page of any type.
```